src/main/java/eu/dissco/core/translator/domain/BioCasePartResult.java

```diff
@@ -0,0 +1,5 @@
+package eu.dissco.core.translator.domain;
+
+public record BioCasePartResult(boolean finished, boolean exception) {
```

Comment:
Maybe something for the next PR, but I'd suggest instead of having a `boolean exception` here, use `Optional<Throwable> e`. That way we can couple the exception that occurred with the biocasePartResult more easily.

Reply:
I don't know if this is necessary. We log the exception and then we shut down the job. Let's change this when we actually need the exception, the change shouldn't be hard to implement.